### ISSUE TITLE
pmh.h has a variable name that collides with Qt keywords

### ISSUE THREAD

**Issue by jpan127** [serge-sans-paille/frozen#61]
https://github.com/serge-sans-paille/frozen/blob/master/include/frozen/bits/pmh.h#L197

The variable name `slots` will fail compilations with any Qt application.  Would it be possible to change it to something else?

Compiled with clang 6.0, -std=c++17, Ubuntu 16.04.

```c++
../include/frozen/bits/pmh.h:197:7: warning: declaration does not declare anything [-Wmissing-declarations]
      cvector<std::size_t, decltype(step_one)::bucket_max> slots;
      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../include/frozen/bits/pmh.h:199:19: error: expected expression
      while (slots.size() < bsize) {
                  ^
../include/frozen/bits/pmh.h:200:48: error: expected expression
        auto slot = hash(key(items[bucket[slots.size()]]), static_cast<size_t>(d.value())) % M;
                                               ^
../include/frozen/bits/pmh.h:200:48: error: expected ']'
../include/frozen/bits/pmh.h:200:42: note: to match this '['
        auto slot = hash(key(items[bucket[slots.size()]]), static_cast<size_t>(d.value())) % M;
                                         ^
../include/frozen/bits/pmh.h:202:59: error: expected expression
        if (H[slot] != UNUSED || !all_different_from(slots, slot)) {
                                                          ^
../include/frozen/bits/pmh.h:203:16: error: expected expression
          slots.clear();
               ^
../include/frozen/bits/pmh.h:208:14: error: expected expression
        slots.push_back(slot);
             ^
../include/frozen/bits/pmh.h:215:10: error: an attribute list cannot appear here
        H[slots[i]] = bucket[i];
         ^~~~~~~~~~
1 warning and 7 errors generated.
```

**Comment by serge-sans-paille** [comment]
#63 should fix that. Can you confirm?

**Comment by jpan127** [comment]
Took a look and it should, I tested with `slts` earlier today.

I don't think `bucket_slots` is a keyword but I can try when I'm in the office tomorrow.

Thanks!

**Comment by serge-sans-paille** [comment]
Fixed by #62
